### PR TITLE
[HB-5094] Adding CONTRIBUTING.md File

### PR DESCRIPTION
**Pull request by SCastanedaMunoz**

### CODE REVIEW

**Review by pleasesavemycat — APPROVED** [review]
lgtm

**Review by kushG — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Follow the Android developers [Kotlin style guide](https://developer.android.com/kotlin/style-guide).
+
+### Swift
+
+Follow Google's [Swift style guide](https://google.github.io/swift/).
+
+### C#
+
+Follow Microsoft's [C# coding conventions guide](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions).
```
Comment (cb-jpadilla) [inline CONTRIBUTING.md:75]: EOL.

**Review by cb-jpadilla — COMMENTED** [review]
We need to add an EOL.

**Review by cb-jpadilla — APPROVED** [review]
Lgtm

**Pull request merged**